Code/GraphMol/SmilesParse/SmilesWrite.h

```diff
@@ -103,7 +103,7 @@ RDKIT_SMILESPARSE_EXPORT std::string MolToSmiles(
 RDKIT_SMILESPARSE_EXPORT std::string MolToSmiles(
     const ROMol &mol, const SmilesWriteParams &params);
 
-//! \brief returns canonical SMILES for a molecule
+//! \brief returns SMILES for a molecule, canonical by default
```

Comment:
This might have snuck in accidentally, not that more docs are bad!

Reply:
I came across it whilst fixing this bug, and it didn't seem worth a separate PR.